ci/ci.sh

```diff
@@ -9,7 +9,7 @@ function initializeTestEnv() {
 	cat <<EOT >> /tmp/parameters.json
 {
         "cas-type": "jiva",
-        "replicaCount": "1"
+        "policy": "example-jivavolumepolicy"
```

Comment:
Please add one more test case to test the same without policy

Reply:
Without the policy, the default replicationfactor will be 3 which cannot be tested on a single node.